AwsCryptographicMaterialProviders/dafny/AwsCryptographyKeyStoreAdmin/src/DescribeMutation.dfy

```diff
@@ -110,4 +112,34 @@ module {:options "/functionSyntax:4" } DescribeMutation {
       Yes := description);
     return Success(Types.DescribeMutationOutput(MutationInFlight := inFlight));
   }
+
+  const TRUST_STORAGE_str := "Trust Storage"
+  const KMS_SYM_ENC_str := "KMS Symmetric Encryption"
+  const UNKOWN_str := "Unknown"
+
+  function SystemKeyDescription(
+    MutationCommitment: KeyStoreTypes.MutationCommitment
+  ): (output: string)
+    ensures
+      && MutationCommitment.CiphertextBlob == SystemKeyHandler.TRUST_STORAGE_UTF8_BYTES
+      ==>
+        output == TRUST_STORAGE_str
+    ensures
+      && MutationCommitment.CiphertextBlob != SystemKeyHandler.TRUST_STORAGE_UTF8_BYTES
+      && KMS.Types.IsValid_CiphertextType(MutationCommitment.CiphertextBlob)
+      ==>
+        output == KMS_SYM_ENC_str
+    ensures
+      && MutationCommitment.CiphertextBlob != SystemKeyHandler.TRUST_STORAGE_UTF8_BYTES
+      && !KMS.Types.IsValid_CiphertextType(MutationCommitment.CiphertextBlob)
+      ==>
+        output == UNKOWN_str
+  {
+    if MutationCommitment.CiphertextBlob == SystemKeyHandler.TRUST_STORAGE_UTF8_BYTES
+    then TRUST_STORAGE_str
+    else
+    if KMS.Types.IsValid_CiphertextType(MutationCommitment.CiphertextBlob)
+    then KMS_SYM_ENC_str
+    else UNKOWN_str
```

Comment:
This is a disappointing outcome from the Dafny formatter.

Reply:
what in tarnation is this formatting?